src/components/visualizations/chart/Chart.tsx

```diff
@@ -37,7 +37,7 @@ export default class Chart extends React.Component<IChartProps> {
         domProps: {},
     };
 
-    private chart: Highcharts.ChartObject;
+    private chart: Highcharts.Chart;
```

Comment:
_Putting the comment here because I cannot give a comment to changed screenshots:_
I see many of the screenshot change axis label zero from `0k` -> `0` (or `0M` -> `0`) (for example https://github.com/gooddata/gooddata-react-components/pull/1018/files?short_path=3b80de1#diff-3b80de1549133ab7588c4e40ba9be39e)

Please just check with designers/PM that this is ok for them.

Also, I'm not sure if it's necessary to change screenshot image permissions from `644`  to `755` but I don't see this is a big deal so you can safely keep it as you have it.

Reply:
- 0k -> 0: accepted at Issues > Chart > 6 [here](https://docs.google.com/document/d/1h8gYLw6dmQErxN_Ab5Si_aDX9vyW4uAbJCyMpDHuCX4/edit#heading=h.d0yuafi3owim)
- Permissions 644 -> 755 i will turn it back

Reply:
Permissions of screenshots have set to 644.